bump-bot/bump_bot.py

```diff
@@ -34,11 +34,13 @@ async def reaction_changed(payload: discord.RawReactionActionEvent, added: bool)
 	
 	message = await message_cache.get_message(payload.message_id, channel)	
 	if message:
-		user = discord.utils.get(client.users, id=payload.user_id)
+
+		# reaction_removed events have no "member", so we grab them via the user_id
+		member = discord.utils.get(client.get_all_members(), id=(payload.member.id if payload.member is not None else payload.user_id))
```

Comment:
I am not sure if get_all_members() could be sus in terms of API requests and such. It seemed to work when doing debugging.

Reply:
Nah, as per documentation, this is equivalent to code that only uses cached stuff and returns all members on all servers. But this would lead to problems when the bot is on multiple servers, better just use channel.guild.members (I doubt all of the other code works well with multiple servers but we can start somewhere)

Reply:
Guild also has get_member which might be nicer